lib/airrecord/table.rb

```diff
@@ -208,11 +208,11 @@ def client
 
     def validate_key(key)
       return true unless key.is_a?(Symbol)
-      raise Error, <<~MSG
```

Comment:
This squiggly Heredoc `<<~` was the only 2.2-incompatible syntax in the codebase. Now that it's gone, we can officially support Ruby 2.2.